api/types/types.proto

```diff
@@ -2689,6 +2689,11 @@ message WindowsDesktopServiceSpecV3 {
     string TeleportVersion = 2 [ (gogoproto.jsontag) = "teleport_version" ];
 }
 
+message WindowsDesktopFilter {
```

Comment:
Any particular reason this is in types.proto instead of authservice.proto?

Reply:
WebSessionFilter and SemaphoreFilter were all in types.proto so it seemed most consistent to keep the filter types in there.